bedrock/firefox/templates/firefox/new/conformity-not-default/scene1.html

```diff
@@ -0,0 +1,48 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% extends "firefox/new/fx-lifestyle-base.html" %}
+
+{% block page_css %}
+  {{ super() }}
+  {% stylesheet 'firefox_new_conformity_not_default' %}
+{% endblock %}
+
+{% block head_content %}
+  <h2>
+    You do you.
+  </h2>
+
+  <p>
+    We get what it’s like to be different. We’re the only non-corporate, non-conformist browser. We celebrate everybody’s right to break the mold, buck the trends, and demand better than the default.
```

Comment:
@craigcook @FrancescoSTL per legal's comment we're replacing one word. Can you replace with: 

We get what it’s like to be different. We’re the only non-profit, non-conformist browser. We celebrate everybody’s right to break the mold, buck the trends, and demand better than the default.